snappi_ixnetwork/vport.py

```diff
@@ -308,15 +312,26 @@ def _set_location(self):
         # calling little bit costly operation? Otherwise we can't handle same port config for
         # multiple run (_set_card_resource_mode reset to card level). Also some ports are
         # handling multiple speed ('novusFourByTwentyFiveGigNonFanOut' and 'novusFourByTenGigNonFanOut')
-        imports = []
+
+        # todo: restructure old module to getChassisWithDetailedResouceGroupsInfo
+        is_uhd = False
+        for port in self._api.snappi_config.ports:
+            location = port.location
+            if '/' in location:
```

Comment:
use chassisobj.chassisType from restpy instead of if '/' from location to set is_uhd. getting from chassis is more accurate than from location format passed

Reply:
I also think that as well. But we have several other module also follow same UHD notation say Star. Some of them also applicable in chassis as well. So port may come for them. Then customer may use old and new format card in same test.
Anyway, please let me know consider those use case. I will update accordingly.

Reply:
In that case we will analyze more on the notations and come up with the common approach later.
for now we will stick to your approach.

Reply:
Thanks!